web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html

```diff
@@ -5,7 +5,7 @@
       [tooltipText]="config.inputFilePath.description"
     ></info-label>
     <app-folder-browser
-      [startPath]="step.options.input_file_path"
+      [startPath]="step.options.inputFilePath"
```

Comment:
To avoid conflicts I suggest rollback ingest options. My PR will take care of that.